providers/tests/celery/executors/test_celery_kubernetes_executor.py

```diff
@@ -46,9 +46,6 @@ def test_is_production_default_value(self):
     def test_serve_logs_default_value(self):
         assert not CeleryKubernetesExecutor.serve_logs
 
-    def test_is_single_threaded_default_value(self):
-        assert not CeleryKubernetesExecutor.is_single_threaded
```

Comment:
Note: I've left the property on the class though, so that the provider will continue to work with Airflow 2.x